internal/proxy/plan_parser.go

```diff
@@ -72,8 +72,17 @@ func (optimizer *optimizer) Exit(node *ant_ast.Node) {
 		floatNodeRight, rightFloat := node.Right.(*ant_ast.FloatNode)
 		integerNodeRight, rightInteger := node.Right.(*ant_ast.IntegerNode)
 
+		// Check IdentifierNodes
+		identifierNodeLeft, leftIdentifier := node.Left.(*ant_ast.IdentifierNode)
+		identifierNodeRight, rightIdentifier := node.Right.(*ant_ast.IdentifierNode)
+		funcName, err := getFuncNameByNodeOp(node.Operator)
```

Comment:
What if the `node.Operator` is not of arithmetic operator?

Reply:
You're right, I have moved this inside each operation that needs it.